env/composite_env_wrapper.h

```diff
@@ -0,0 +1,1230 @@
+// Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <rocksdb/env.h>
```

Comment:
I think we mostly use `#include "rocksdb/env.h"`. The difference is where to start the search. This is OK, but maybe it's good idea to be consistent across the codebase.